### PR TITLE
Create an index page with links to signup and login forms

### PR DESCRIPTION
**Pull request by evamaina**
### What does this PR do?
Adds a landing with links to signup and login form
### Description
The landing page is the first page the visitors see before signing up or logging in. It has a link for signup and login form as well as few details about the ride.
### Tasks achieved in this PR
* index page
* signup page
* login page
### How should this be manually tested?
Clone this repo and test locally on your computer.
### Background Context?
The UI has no core functionality
### Resources and libs
[Pivotal tracker board](https://www.pivotaltracker.com/n/projects/2179554)
### Screenshots
index
index2

signup

login

### CODE REVIEW

**Comment by evamaina** [comment]
.

**Comment by TheSteelGuy** [comment]
Nice one though I see there are repeated fields there. I guess you don't want it that way

**Comment by JoshuaOndieki** [comment]
👍 Great choice of colors.
👍 Responsive layout.
- I think it would be nice if all images in the slider had the same dimensions.
- Your login page seems to have 2 **signup** buttons
- Since your password field in the login page doesn't have a label, using asterisks as placeholder isn't a good decision.
- Your 'home' link is broken, fix it.

File: UI/login.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<html>
+<head>
+<meta name="viewport" content="width=device-width,initial-scale=1.0"/>
+<link href="css/style.css" type="text/css" rel="stylesheet"/>
```
Comment (JoshuaOndieki) [inline UI/login.html:4]: GIve your stylesheets descriptive file names

**Comment by Ogollah** [comment]
Looks nice.

**Comment by evamaina** [comment]
Thanx will work on that

File: UI/signup.html [hunk 2]
```diff
@@ -21,15 +23,15 @@
...
-					<p><input type="text" placeholder="Enter First Name" required="required"></p>	
-					<p><input type="text" placeholder="Enter Last Name" required="required"></p>
-					<p><input type="text" placeholder="Enter Username" required="required"></p>
-					<p><input type="email" placeholder="Enter Email" required="required"></p>
-					<p><input type="number" placeholder="Enter Phone Number" required="required"></p>
-					<p><input type="password" placeholder="Enter Password" required="required"></p>
-					<p><input type="password" placeholder="Confirm password" required="required"></p>
-					<p><input type="submit" value="SIGNUP"></p>
+				<form method="post" action="login.html" name="signUp" onsubmit="return validateForm()"">
+					<p><input type="text" placeholder="Enter First Name" name="Fname" id="Fname"></p>
```
Comment (JoshuaOndieki) [inline UI/signup.html]: Is the **p** tag necessary around the input fields?

File: UI/login.html (hunk 1 above)
Comment (evamaina) [reply UI/login.html:4]: noted, thanx

**Comment by xcixor** [comment]
Awesome, and you can add some spacing to the form placeholders so that they dont appear too close to the edges

**Comment by Xerrex** [comment]
The color design are quite good. I  know you have a purpose for the many signup fields but consider consolidating them to ease Sign up to your app. From the screenshots you have repeated fields.

**Pull request merged**